src/cuga/backend/server/manage_routes/config_routes.py

```
@@ -0,0 +1,437 @@
+                try:
+                    from cuga.backend.knowledge.config import KnowledgeConfig
+
+                    kb_config = KnowledgeConfig.coerce_and_validate(knowledge_cfg)
+                    _initializer = getattr(state, "initialize_knowledge_engine", None)
+                    if _initializer:
+                        await _initializer(state, kb_config)
+                        engine = state.knowledge_engine
+                        logger.info("Knowledge engine created via publish (was disabled at startup)")
+                    else:
+                        logger.warning("Knowledge engine initializer not available")
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)